### ISSUE TITLE
pretraining distribution

### ISSUE THREAD

**Issue by tranh215980** [hms-dbmi/CHIEF#20]
Dear authors,

What is pretraining distribution of CTransPath and CHIEF? Is new CHIEF CTransPath model the same as previous one in Medical Image Analysis?

**Comment by Dadatata-JZ** [comment]
Hi Tran, 

As you may know, the pre-training was done using a self-supervised approach, utilizing these 60,530 slides. The aim of this pre-training was to build CHIEF with a "comprehensive" understanding of histology images.

**Comment by Dadatata-JZ** [comment]
@Eshay14 thanks for your attempt to help explain @tranh215980 's questions. 

To clarify, our image encoder utilizes the SELF-SUPERVISED CTransPath backbone to extract histopathology image feature representations at the tile level. For answers to the remaining questions, please refer to our response in the other thread.

https://github.com/hms-dbmi/CHIEF/issues/21#issuecomment-2354153604

**Comment by Dadatata-JZ** [comment]
@Eshay14 
Absolutely! Your understanding of the encoder is mostly correct. However, please note that the texts are inputs during the pre-training phase, not ground truth labels. The supervised labels are binary: 'cancer' and 'non-cancer.' 

During inference (tumor origin prediction, EXTENDED FIGURE 1), these texts are no longer passed into CHIEF. Following evaluation standard in computational pathology, we reported the held-out (internal) set from TCGA for reference, while CPTAC serves as the external, independent validation set, which was never used in the pre-training process.

Cheers,
